RELEASE.md

```diff
@@ -11,6 +11,7 @@
 # Upcoming Release 0.18.6
 
 ## Major features and improvements
+* Added new Kedro CLI `kedro jupyter-init` to setup Jupyter Kernel for Kedro.
```

Comment:
```suggestion
* Added new CLI command `kedro jupyter init` to setup a Jupyter kernel for Kedro.
```

Reply:
I totally missed the comments about the command names, I might slightly prefer `setup`.

Reply:
> would be nice to make a note of this in the docs, maybe [here](https://kedro.readthedocs.io/en/stable/notebooks_and_ipython/kedro_and_notebooks.html#useful-to-know). e.g. on a managed service like Databricks you can run kedro jupyter init and then select the kernel from the databricks notebook rather than needing to explicitly run %load_ext kedro.ipython

@AntonyMilneQB Can we actually do this? Can you select a custom Jupyter kernel on Databricks? It could still work for managed JupyterHub though.

Reply:
Hmm, good question. I thought you could select a custom Jupyter kernel on databricks but I might just be remembering wrongly. So yeah, we should check before recommending that... Maybe I was thinking of Sagemaker instead? I'm sure you can select custom kernel there.